pkg/provisioningv2/rke2/installer/installer.go

```diff
@@ -69,6 +71,9 @@ func LinuxInstallScript(token string, envVars []corev1.EnvVar, defaultHost strin
 		}
 	}
 	ca := systemtemplate.CAChecksum()
+	if v, ok := ctx.Value(tls.InternalAPI).(bool); ok && v {
```

Comment:
For reviewers: this is copied from here: https://github.com/rancher/rancher/blob/60275b1f66d2e2c0ae40a9edaa7ca874ac2c23b6/pkg/cacerts/handler.go#L36